target_chains/stylus/contracts/pyth-receiver/src/structs.rs

```diff
@@ -0,0 +1,17 @@
+use alloc::vec::Vec;
+use stylus_sdk::{prelude::*, storage::{StorageU16, StorageU64, StorageI32, StorageI64, StorageFixedBytes}};
+use stylus_sdk::alloy_primitives::{U64, I32, I64};
+
+// PriceInfo struct storing price information
+#[storage]
+pub struct PriceInfoStorage {
+    pub publish_time: StorageU64,
+    pub expo: StorageI32,
+    pub price: StorageI64,
+    pub conf: StorageU64,
+    pub ema_price: StorageI64,
+    pub ema_conf: StorageU64,
+}
+
+// PriceInfo struct storing price information
+pub type PriceInfoReturn = (U64, I32, I64, U64, I64, U64);
```

Comment:
I forgot why we couldn't make this a type. it makes it very difficult to understand the response.